README.md

```diff
@@ -74,26 +74,49 @@ We use the [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/late
 to store secrets for this project.  An AWS best practice is to create secrets
 with a unique ID to prevent conflicts when multiple instances of this project
 is deployed to the same AWS account.  Our naming convention is
-`<cfn stack name>/<environment id>/<secret name>`.  An example is `myapp/dev/MySecret`
+`<cfn stack name>/<environment id>/<secret name>`.  The template in this repo' uses
+the secret name, `ecs`, so an example Secrets Manager name is `myapp/dev/ecs`.
+
+
+## Deployment from GitHub to AWS
+
+To allow the GitHub action in your repository to deploy infrastructure to AWS, submit a
+PR like [this](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/771/files),
+customizing the `StackName`, `Repositories` (name), and the `Account`(s) to 
+in which the infrastructure will be deployed.
+
+Once the PR is merged, an IAM role will be created in each AWS account specified in the PR.  
+The ARN for the role will go into the `role-to-assume` field of `aws-deploy.yml`.  
+NOTE:  To separately deploy into different AWS accounts (say, `dev`, `test`, and `prod`), 
+create separate GitHub branches and customize `aws-deploy.yml` in each branch with the ARN 
+of the role for the target AWS account.
```

Comment:
you could just reference the docs from organizations-infra repo.. https://github.com/Sage-Bionetworks-IT/organizations-infra/tree/master/org-formation/650-identity-providers

Reply:
Done.